master/internal/db/postgres_test_utils.go

```diff
@@ -326,3 +328,51 @@ func (db *PgDB) MustExec(t *testing.T, sql string, args ...any) sql.Result {
 	require.NoError(t, err, "failed to run query")
 	return out
 }
+
+// MockWorkspaces creates as many new workspaces as in workspaceNames and
+// returns a list of workspaceIDs
+func MockWorkspaces(workspaceNames []string, userID model.UserID) ([]int, error) {
+	ctx := context.Background()
+	var workspaceIDs []int
+	var workspaces []model.Workspace
+	workspaceIDMap := map[string]bool{}
+
+	for _, workspaceName := range workspaceNames {
+		workspaces = append(workspaces, model.Workspace{
+			Name:   workspaceName,
+			UserID: userID,
+		})
+		workspaceIDMap[workspaceName] = true
+	}
+
+	_, err := Bun().NewInsert().Model(&workspaces).Exec(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	workspaces = []model.Workspace{}
+	err = Bun().NewSelect().Model(&workspaces).
+		Where("name IN (?)", bun.In(workspaceNames)).
```

Comment:
It seems like if there are workspaces in the DB with the same workspace names we just created we don't need to worry about them?